python/invoke_release/tasks.py

```diff
@@ -696,6 +696,22 @@ def _create_branch(verbose, branch_name):
     _verbose_output(verbose, 'Done creating branch {}.', branch_name)
 
 
+def _create_local_tracking_branch(verbose, branch_name):
+    _verbose_output(
+        verbose,
+        'Creating local branch {branch} set up to track remote branch {branch} from \'origin\'...',
```

Comment:
FWIW, our styleguide permits using double quotes to prevent having to escape single quotes, but I'm not going to hold it up over this.